### ISSUE TITLE
Get music-suite into stackage

### ISSUE THREAD

**Issue by hanshoglund** [music-suite/music-suite#15]
As we're moving to build everything with Stack #14, a reasonable goal is to eventually get Music Suite into LTS.

AFAIU what needs to be done is:
- Before making a release to Hackage, make sure it builds with current nightly, i.e. https://www.stackage.org/nightly
- One hour after making the release, open a PR to Stackage as per below

https://github.com/fpco/stackage/blob/master/MAINTAINERS.md

**Comment by OndrejPopp** [comment]
Ok, it's three years later now. 
Hello Hans,
I am planning to use your software for a musical project. To get started 
I have made a version that compiles with the current stackage lts.
Because the music suite libraries are quite dependent on each other I have merged them all into music-suite. Here, https://github.com/OndrejPopp/music-suite/tree/merge-v1.9.2. There is now only one dependency left, and that is monadplus. I left that one out because monadplus does not depend on the music-suite libraries. Next, monadplus needs a patch which is already a pull request from asr "Added Subgroup instance" with that one everything compiles ok now with stack build. I have decided not to make a pull request of this merge because you do not seem to be that active with this software anymore. So this is just a message to let you know about the merge.

Thanks for writing this software. I saw that you studied at a musical conservatory. Which is cool, because your software carries a lot of your musical know how which is very useful to me.

Kind regards,
Ondrej
